### ISSUE TITLE
Consider having Cmd+P à la Sublime/Atom/Spotlight search instead of a dumb "find in this page"

### ISSUE THREAD

**Issue by victorporof** [mozilla/tofino#219]
At the moment we can search by ctrl/cmd+f in page, and then we'd have an awesomebar-like thing for searching bookmarks, history etc. It'd be nice to unify those and have a global "search for anything", while still staying true to the mockups maybe. Ping: @phlsa.

**Comment by jsantell** [comment]
Vivaldi has [something similar](http://vivalditips.com/efficiency/quick-commands), I like this idea, but wonder where the line between awesome/urlbar and the spotlights search is

**Comment by phlsa** [comment]
Yeah, having two url/search/command bars sounds like it can get pretty confusing. But I'm absolutely on board with the idea of doing more through whatever bar we get than just navigating.
